.pylintrc

```diff
@@ -38,7 +38,7 @@ load-plugins=linter_plugin
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=fixme,locally-disabled,abstract-class-not-used
+disable=fixme,locally-disabled,abstract-class-not-used,duplicate-code
```

Comment:
Could you refactor the code instead of disabling this warning? Unfortunately, I don't think it can be disabled globally, so it has a big impact on the rest of the project.